INSTALL.rst

```diff
@@ -4,6 +4,18 @@ INSTALLING PaRSEC
 
 .. contents:: Table of Contents
 
+
+TL;DR
+=====
+
+.. code:: bash
+
+  mkdir builddir && cd builddir
+  ${srcdir}/configure --with-hwloc --with-mpi --disable-debug --prefix=$PWD/install
+  make install
+  mpiexec -n 8 examples/ex00
+
```

Comment:
Should we document available CMake options? I always find it tedious figuring out what options are available, configure does a much better job at that.

Reply:
I'd say no, because we will make a poor job of updating the documentation when we do change them and that will drift over time.